internal/jimm/juju/model.go

```diff
@@ -251,11 +253,15 @@ func (j *JujuManager) ModelInfo(ctx context.Context, user *openfga.User, mt name
 		return nil, errors.E(op, err)
 	}
 
-	if ok, err := user.IsModelReader(ctx, mt); !ok || err != nil {
+	relation := user.GetModelAccess(ctx, mt)
```

Comment:
I'd prefer to avoid these calls that get the access level, instead opting to check if the user has the minimum required access level.
In this case for ModelInfo, we could check if the user is a ModelReader.

Reply:
the thing is that the user will see different level of detail depending on their access level